### PR TITLE
random: `lcg_value()` is deprecated

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

**Comment by Girgias** [comment]
Please add a `deprecated` attribute in `versions.xml` so that it is listed separately in the sidebar.

File: reference/random/functions/lcg-value.xml [hunk 1]
```diff
@@ -46,6 +51,31 @@
...
+       <entry>8.4.0</entry>
+       <entry>
+        This function has been deprecated.
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </informaltable>
+  </para>
+ </refsect1>
```
Comment (Girgias) [inline reference/random/functions/lcg-value.xml:77]: Useless wrapping <para> tag.

Comment (TimWolla) [reply reference/random/functions/lcg-value.xml:77]: Referring to the `<para>` or which one?

Comment (Girgias) [reply reference/random/functions/lcg-value.xml:77]: Indeed the para one yes

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**